README.md

```diff
@@ -42,6 +42,14 @@ make run
 make run-backend 
 ```
 
+### Expose saleor-app-checkout
+```shell
+make tunnel
```

Comment:
This "locks" to serving only checkout, but storefront soon will be the app and can be tunneled as well. So in this case script could be `make checkout tunnel` or something like this

Reply:
Fixed